docs/src/components/Card/card.module.css

```diff
@@ -106,11 +106,13 @@
   height: 80px;
   display: flex;
   justify-content: center;
+  align-self: stretch;
```

Comment:
Since we limit the width to 80% now, i think this might need to be centered

```suggestion
  align-self: center;
```

<img width="926" alt="Screenshot 2024-12-04 at 8 51 25 AM" src="https://github.com/user-attachments/assets/a1cfc77d-8685-4b2d-b3d6-62de01ccda8b">

Reply:
@daniellok-db this is weird, I don't see this misalignment in my local. Is there any style override set in dev tools for this screenshot?